plugin.onedrive/addon.py

```diff
@@ -0,0 +1,210 @@
+#-------------------------------------------------------------------------------
```

Comment:
same comment as the google drive addon please consider to keep this file to a minimum in the future

Reply:
will do